### DISCUSSION TITLE
Even though my data is undirectional graph can the explainer still give directed graph exaplantion

### DISCUSSION THREAD

**Discussion by bojack-horseman91** [pyg-team/pytorch_geometric#9261] [Q&A]
I have checked that all my data is undirectional using is_directed() function and found none are directional and yet when I use GNNExxplainer to produce an explanation I get a directed graph visualized when I run explanation.get_explanation_subgraph().visualize_graph("path.png"). Can you tell me what's wrong? 
P.S: I have used PNAConv as my GNN then feed it to a Fully connected layer

**Comment by rusty1s** [comment]
We visualize a directed graph since the scores for an edge may still be different depending on its direction. Imagine you have a one-layer GNN, then you would expect that all outgoing edges from a seed node have importance score ~ 0 (since they do not contribute to the final output), while incoming edges have an importance score > 0.

**Comment by bojack-horseman91** [comment]
So for interpretation purpose how do we percieve these directions? Because
my edges are representing relationship between two genes how do i make
inference  from the direction of the graph ?

**Reply by rusty1s** [reply]
I would just interpret them as information flow. One edge updates node A by information from node B, and one edge updates node B by information from node A.

> how do i make inference  from the direction of the graph ?

Can you clarify? I don't understand yet.

**Comment by bojack-horseman91** [comment]
Well I was hoping to get some biological inference because I had set the
edge attribute to be co-methylation ,which is a fancy way of saying
correlation between two genes. How do i interpet the direction in a
biological terms?
